test/src/main/configurations/UTIL/Larva/xsl/CreateMail.xsl

```diff
@@ -2,7 +2,7 @@
 <xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" version="2.0">
 	<xsl:output method="xml" indent="yes" cdata-section-elements="message" />
 
-	<xsl:param name="hostname"/>
+	<xsl:param name="hostname">asdf435sdf</xsl:param>
```

Comment:
Hier een herkenbare naam gebruiken, zoals `fakehostname` of `host.under.test`.
Dat bespaart je uitleg zoals:
```
[Gisteren 09:52] Ualit, T. (Tarik)
asdf435sdf is de servernaam
```